docs/pages/upgrade.mdx

```diff
@@ -14,6 +14,13 @@ Version 5: Rework of our foundations
 - The `quaternary` variant is now named `ghost`.
 - The icon size now adjusts to the size of the button.
 
+#### Tag
+
+- We have adjust colors and spacing.
```

Comment:
We have adjust**ed** (again, not a important fix sorry)

Reply:
Well seen 😇 🙏